include/rear-panel/core/tasks.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+#include "common/core/freertos_message_queue.hpp"
+#include "common/core/freertos_timer.hpp"
+#include "eeprom/core/hardware_iface.hpp"
+#include "eeprom/core/task.hpp"
+#include "i2c/core/hardware_iface.hpp"
+#include "i2c/core/messages.hpp"
+#include "i2c/core/tasks/i2c_poller_task.hpp"
+#include "i2c/core/tasks/i2c_task.hpp"
+#include "i2c/core/writer.hpp"
+
+namespace rear_tasks {
+
+/**
+ * Start gripper tasks.
```

Comment:
let's go ahead and clean up the copy/paste artifacts before doing the first commit